src/sentiment.py

```diff
@@ -22,10 +22,13 @@ def __init__(self, file):
         return
 
     def analyze(self, text):
-        return self.analyzer.analyze(
+        response = self.analyzer.analyze(
             text=text,
-            features=Features(
-                entities=EntitiesOptions(emotion=True, sentiment=True, limit=2),
-                keywords=KeywordsOptions(emotion=True, sentiment=True,
-                                         limit=2))).get_result()
+            features=Features(sentiment=SentimentOptions(document=True))).get_result()
 
+
+        return (float(response["sentiment"]["document"]["score"]) + 1) * 5
```

Comment:
why add 1 and multiply by 5? what does the score return?

Reply:
ah should've been clearer. The API will score in the range -1 to 1.  But in our paper we said its gonna be from 0 to 10, so I did some direct mapping lol cause its the same thing.